UnityProject/Assets/Scripts/UI/UIManager.cs

```diff
@@ -4,187 +4,208 @@
 
```

Comment:
I cant check this file, due to all lines being flagged changed due to replacing tabs with spaces.